lib/storage/bucket.js

```diff
@@ -288,8 +306,51 @@ Bucket.prototype.getMetadata = function(callback) {
 };
 
 /**
+ * Restrict access to your bucket's contents to only project owners.
+ *
+ * @param {function=} callback - The callback function.
+ *
+ * @example
+ * bucket.makePrivate(function(err) {});
+ */
+Bucket.prototype.makePrivate = function(callback) {
+  callback = callback || util.noop;
+
+  this.setMetadata({
+    predefinedAcl: 'private'
+  }, function(err) {
+    callback(err || null);
+  });
+};
+
+/**
+ * Make the files in your bucket publicly readable.
+ *
+ * @param {function=} callback - The callback function.
+ *
+ * @example
+ * bucket.makePublic(function(err) {});
+ */
+Bucket.prototype.makePublic = function(callback) {
+  callback = callback || util.noop;
+
+  this.setMetadata({
+    predefinedAcl: 'publicRead'
```

Comment:
From the [docs](https://cloud.google.com/storage/docs/accesscontrol#predefined-acl):

> Gives the bucket or object owner OWNER permission and gives all users, both authenticated and anonymous, READER permission. When you apply this to an object, anyone on the Internet can read the object without authenticating. When you apply this to a bucket, anyone on the Internet can list objects without authenticating. 

If by "list objects", they mean read the objects contents, then yes.